lib/ansible/modules/crypto/openssl_publickey.py

```diff
@@ -59,6 +59,11 @@
             - The passphrase for the private key.
         type: str
         version_added: "2.4"
+    comment:
+        description:
+            - A comment to append to the public key
+        type: str
+        version_added: "2.8"
```

Comment:
```suggestion
        version_added: "2.9"
```
No new features are accepted for Ansible 2.8.